samtranslator/internal/schema_source/aws_serverless_graphqlapi.py

```diff
@@ -55,6 +55,25 @@ class ResolverCodeSettings(BaseModel):
     FunctionsFolder: Optional[str]
 
 
+class Sync(BaseModel):
+    ConflictDetection: PassThroughProp
+    ConflictHandler: Optional[PassThroughProp]
+    LambdaConflictHandlerArn: Optional[PassThroughProp]
+
+
+class Function(BaseModel):
+    DataSource: Optional[str]
```

Comment:
`Optional[Literal["NONE"]]`?

Reply:
`DataSource` also takes relative id of datasources defined in the same `Serverless::GraphQLApi ` resource.